src/customTypes.d.ts

```diff
@@ -80,3 +80,29 @@ interface Window {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   gtag: any;
 }
+
+export type OneRepScanResult = {
```

Comment:
I think it's probably good to limit this file to limit this file to providing [ambient type definitions](https://www.typescriptlang.org/docs/handbook/modules.html#working-with-other-javascript-libraries), and (especially now that we can write TS ourselves) define types returned by our own functions next to the code where data matching that shape originates.

I'll also note that there is already a type for the API response that OneRep provides for Scan Results: https://github.com/mozilla/blurts-server/blob/75e7d70988ddc122e2882feaa69aa807d0218819/src/external/onerep.js#L249-L264

If this is defines the structure of database columns that we define ourselves, it's probably a good idea not to name it `OneRep`. In fact, Knex has a more standard way to define types for it table rows - let me create a separate PR to demonstrate that.

Reply:
https://github.com/mozilla/blurts-server/pull/3167

Reply:
OK I ended up extracting the types from the code you landed a while back (thanks for pointing that out!) and added its own `.d.ts` file, I might not totally be understanding where you want to go wrt types in Knex but we can discuss further. I stopped using `OneRep` in the type defs.